src/main/java/com/comphenix/protocol/reflect/cloning/BukkitCloner.java

```diff
@@ -88,6 +89,11 @@ private static void fromManual(Supplier<Class<?>> getClass, Function<Object, Obj
 			fromManual(ComponentConverter::getBaseComponentArrayClass, source ->
 					ComponentConverter.clone((BaseComponent[]) source));
 		} catch (Throwable ignored) { }
+
+		try {
+			fromManual(AdventureComponentConverter::getComponentClass, source ->
+					AdventureComponentConverter.clone((Component) source));
```

Comment:
Realistically you don't need to cast to Component here, you could just change the clone method below to accept objects